### ISSUE TITLE
Exponential Learning Rate Schedules for Deep Learning

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#1673]
## 一言でいうと

学習率は徐々に下げていく、という通説を覆し指数関数的に上げていく手法の提案。Batch Normalizationが導入されている場合、Weight Decay(≒L2正則化)＋一定率(c)での学習率の減衰+momentumはWeight Decayなし＋指数関数的LR増大+momentumと等価であることを理論的＋実験的に証明している。

### 論文リンク

https://arxiv.org/abs/1910.07454

### 著者/所属機関

Zhiyuan Li, Sanjeev Arora

* Princeton University
* Princeton University and Institute for Advanced Study

### 投稿日付(yyyy/MM/dd)

2019/10/16
